HackSessionHowTo.md

```diff
@@ -1,16 +1,27 @@
-# How to Start a Hack Session
-Anyone can start a Hack Session! It's a good idea to have found at least one other interested member before opening an issue. The issue MUST contain:
+# Hack Sessions
+DevStaff members occasionally organise Hack Sessions on various topics. 
+Org Team will help with great pleasure to organize a Hack Session and create 
```

Comment:
The Org Team

Reply:
Fixed at https://github.com/devstaff-crete/DevStaff-Heraklion/pull/493/commits/29b229b36979eec3c27d5368e338226b287a0f8a